src/ScriptCs.Hosting/ScriptServicesBuilder.cs

```diff
@@ -48,7 +49,7 @@ public ScriptServices Build()
 
             _scriptEngineType = (Type)Overrides[typeof(IScriptEngine)];
 
-            var initDirectoryCatalog = _scriptName != null || _repl;
+            var initDirectoryCatalog = _scriptName != null || _repl || _loadScriptPacks;
```

Comment:
Good point

On Fri, Mar 27, 2015 at 12:26 AM Filip W <notifications@github.com> wrote:

> In src/ScriptCs.Hosting/ScriptServicesBuilder.cs
> <https://github.com/scriptcs/scriptcs/pull/999#discussion_r27279248>:
>
> > @@ -48,7 +49,7 @@ public ScriptServices Build()
> >
> >              _scriptEngineType = (Type)Overrides[typeof(IScriptEngine)];
> >
> > -            var initDirectoryCatalog = _scriptName != null || _repl;
> > +            var initDirectoryCatalog = _scriptName != null || _repl || _loadScriptPacks;
>
> this is still kinda confusing since I can say LoadScriptPacks(false) and
> yet it will override me because I have set a filename. It would be better
> to just check _loadScriptPacks here - this way I can have script
> execution or REPL without initializing script packs if I don't care about
> them in my host
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scriptcs/scriptcs/pull/999/files#r27279248>.
>